### PR TITLE
[MODULAR] Gas Mask Tint Removal

### PR DESCRIPTION
**Pull request by jwc1015**
## About The Pull Request
Changes gas masks so they no longer have screen tint, at all. Effectively reverts #9817 in a modular way.

## How This Contributes To The Skyrat Roleplay Experience
From my knowledge, many people use gas masks for flavor essentially, due to the accessibility of things like the breathing tube implant; the original PR effectively just makes that more difficult.

## Changelog

:cl:
Balance: Gas masks no longer tint/darken your view.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/gasmasktint/code/explorer_gear.dm [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+/obj/item/clothing/mask/gas/explorer/attack_self(mob/user)
+	adjustmask(user)
+
+/obj/item/clothing/mask/gas/explorer/adjustmask(user)
+	..()
+	w_class = mask_adjusted ? WEIGHT_CLASS_NORMAL : WEIGHT_CLASS_SMALL
+
+/obj/item/clothing/mask/gas/explorer/folded/Initialize(mapload)
+	. = ..()
+	adjustmask()
```
Comment (tf-4) [inline modular_skyrat/modules/gasmasktint/code/explorer_gear.dm]:
```suggestion
/obj/item/clothing/mask/gas/explorer
	tint = 0
```

File: modular_skyrat/modules/gasmasktint/code/gasmask.dm [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+
+/obj/item/clothing/mask/gas/driscoll
+	name = "driscoll mask"
+	desc = "Great for train hijackings. Works like a normal full face gas mask, but won't conceal your identity."
+	icon_state = "driscoll_mask"
+	flags_inv = HIDEFACIALHAIR
+	w_class = WEIGHT_CLASS_NORMAL
+	inhand_icon_state = "driscoll_mask"
+	flags_cover = MASKCOVERSEYES
+	tint = 0
```
Comment (tf-4) [inline modular_skyrat/modules/gasmasktint/code/gasmask.dm]:
```suggestion
/obj/item/clothing/mask/gas
	tint = 0

/obj/item/clothing/mask/gas/syndicate
	tint = 0
```

Comment (tf-4) [inline modular_skyrat/modules/gasmasktint/code/gasmask.dm:391]:
```suggestion
/obj/item/clothing/mask/gas
	tint = 0

/obj/item/clothing/mask/gas/syndicate
	tint = 0
```

Comment (tf-4) [inline modular_skyrat/modules/gasmasktint/code/gasmask.dm:391]:
```suggestion
/obj/item/clothing/mask/gas
	tint = 0

/obj/item/clothing/mask/gas/syndicate
	tint = 0
```

**Comment by tf-4** [comment]
Gas masks are not purely flavour, and should not be objectively the best thing to wear on your face under all circumstances. I disagree with this PR.

**Comment by Gandalf2k15** [comment]
The feature was just added and still needs time to settle upstream, incoming PRs will conflict.

**Pull request closed without merging**